src/components/ui/UploadEmailListButton.tsx

```diff
@@ -0,0 +1,33 @@
+import { TextButton } from 'components/ui/Text'
+import EmailFormStore from 'stores/EmailFormStore'
+import ToolTip from 'components/ui/ToolTip'
+import handleFile from 'helpers/handleFile'
+
+export default function ({
+  title = 'Upload an email list (txt, csv, etc...)',
+  small,
+  center,
+}: {
+  title?: string
+  small?: boolean
+  center?: boolean
+}) {
+  return (
+    <TextButton
+      onClick={() =>
+        handleFile(EmailFormStore.setEmailListFromFile.bind(EmailFormStore))
```

Comment:
Looks weird. Do we need `bind` here?

Reply:
Yes, when passing the class into the callback (like here) the context (`this`) is lost, so we need `.bind` here and inside this function, in other case you'll get:
`this is undefined`